index.html

```diff
@@ -444,7 +360,7 @@ <h1>Get Started</h1>
         </footer>
     </section>
 </section>
-    
+<div class="modals-section"></div>
```

Comment:
- [ ] `required`
- [ ] Good job so far 👏 
- [ ] However if I type in Block letters on the `email` input the form is submitting instead of giving me an error message.
- [ ] kindly make the email all lowercase by comparing if the email and its lowercase converted version are the same, ie: email === email.toLowerCase(), utilizing regular expressions or looping through each letter of the email string. 🥂

Reply:
Hey, good Morning. I’m sorry for the inconvenience but I accidentally submitted the wrong branch for review. This is my pop up modal branch and not my JavaScript validation branch. If you could kindly make this review request invalid so I can submit the link to the right branch. Thanks.